app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettingsFragment.kt

```diff
@@ -63,17 +58,7 @@ class WallpaperSettingsFragment : Fragment() {
                                 )
                             )
                         },
-                        onViewWallpaper = { findNavController().navigate(R.id.homeFragment) },
-                        tapLogoSwitchChecked = wallpapersSwitchedByLogo,
-                        onTapLogoSwitchCheckedChange = {
-                            settings.wallpapersSwitchedByLogoTap = it
-                            wallpapersSwitchedByLogo = it
-                            Wallpapers.changeWallpaperLogoToggled.record(
-                                Wallpapers.ChangeWallpaperLogoToggledExtra(
-                                    checked = it
-                                )
-                            )
-                        }
+                        onViewWallpaper = { findNavController().navigate(R.id.homeFragment) }
```

Comment:
nit: another trailing comma spot 😅 apparently I missed them in the original implementation so it would be helpful to get them in now